mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -101,3 +118,101 @@ func ignitionFailure(c cluster.TestCluster) error {
 		return nil
 	}
 }
+
+// Attach a block device which has a pre-existing boot filesystem on it
+func bootfsFailure(c cluster.TestCluster) error {
+
+	// get current path and create tmp dir
+	dirname := "tmp/fakeboot"
+	bootName := "fakeboot"
+	dir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	// Change the current working directory
+	tempDir := os.Chdir(filepath.Join(dir, dirname))
+	if err != nil {
+		return err
+	}
+	fmt.Println("Current dir: ", tempDir)
+
+	createBoot, err := os.Create(bootName)
+	if err != nil {
+		return err
+	}
+
+	// Get the absolute path of the file
+	absPath, err := filepath.Abs(createBoot.Name())
+	if err != nil {
+		fmt.Println("Error getting absolute path:", err)
+		return err
+	}
+	fmt.Println("absPath: ", absPath)
+	// Truncate the file to 1 gigabyte
+	const oneGB = 1 << 30
+	err = os.Truncate(absPath, oneGB)
```

Comment:
We have a direct handle to the opened file, so we can do

```suggestion
	err = createBoot.Truncate(oneGB)
```